cdk/kittyhawk/test/ingress.test.ts

```diff
@@ -1,8 +1,87 @@
 import { Construct } from "constructs";
 import { domainToCertName, removeSubdomain } from "../src";
 import { Application } from "../src/application";
-import { failingTest } from "./utils";
+import { chartTest, failingTest } from "./utils";
 
+/*
+Ingress Port Behavior 
+
+1. NEITHER: Default Behavior
+If `port` IS NOT specified in neither `Application` nor `ingressProps`, 
+continue with default behavior in `Application` and other children objects 
+created. For `Ingress`, the default port is assumed.
+
+2. In Application, Not Ingress: Inherits Application port
+If `port` IS NOT specified in `Application` and `port` IS specified in `ingressProps`, 
+the ingress port value is used for the `Ingress`
+
+3. In Ingress, Not Application: Uses Ingress Port for Ingress
+- If `port` IS specified in `Application` and `port` IS NOT specified in `ingressProps`, 
+the application port value is passed to `Ingress` as well.
+
+4. BOTH: Ingress Port Overrides Application
+- If `port` IS specified in both `Application` and `ingressProps`, the ingress port value
+ overrides the application port specified for the `Ingress`.
+*/
+
+export function buildDefaultIngressChart(scope: Construct) {
+  new Application(scope, "serve", {
+    deployment: {
+      image: "pennlabs/website",
+      tag: "latest",
+    },
+  });
+}
+
+export function buildCustomApplicationPortIngressChart(scope: Construct) {
+  new Application(scope, "serve", {
+    deployment: {
+      image: "pennlabs/website",
+      tag: "latest",
+    },
+    port: 433,
```

Comment:
It doesn't really matter, but the port here (and below) were supposed to be 443, right?